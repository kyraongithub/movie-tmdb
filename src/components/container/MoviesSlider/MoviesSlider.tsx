import type { MovieInterface } from '../../../interfaces/Movie.interface';
import Card from '../../ui/Card';
import LoaderSkeleton from '../../ui/Skeleton';
import type { SliderPropsInterface } from './Slider.interface';

const MoviesSlider = (props: SliderPropsInterface) => {
  const { movieList, link, isLoading, title } = props;

  if (isLoading) {
    return <LoaderSkeleton width={200} height={300} />;
  }

  return (
    <section className="w-full px-4 py-6">
      <div className="flex items-center justify-between mb-4">
        <h2 className="text-xl font-bold  hover:text-primary transition-colors">
          <h1>{title}</h1>
        </h2>
        <a
          href={`/category/${link}`}
          className="text-sm text-gray-400 hover:text-white transition-colors"
        >
          View All
        </a>
      </div>

      <div className="relative">
        <div className="flex overflow-x-scroll space-x-4 px-4 pb-4 scrollbar-hide">
          {movieList?.map((movie: MovieInterface) => (
            <Card
              key={movie.id}
              id={movie.id}
              movie={movie}
              className="min-w-[200px]"
            />
          ))}
        </div>
      </div>
    </section>
  );
};

export default MoviesSlider;
