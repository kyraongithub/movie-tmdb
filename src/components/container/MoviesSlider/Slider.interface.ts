import type { MovieInterface } from '../../../interfaces/Movie.interface';

export interface SliderPropsInterface {
  movieList: MovieInterface[];
  link: string;
  isLoading: boolean;
  title: string;
}
