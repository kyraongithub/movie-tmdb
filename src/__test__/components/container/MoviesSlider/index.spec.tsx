import { render } from '@testing-library/react';
import MoviesSlider from '../../../../components/container/MoviesSlider/MoviesSlider';

describe('MoviesSlider', () => {
  it('render MoviesSlider component', () => {
    const props = {
      movieList: [
        {
          adult: false,
          backdrop_path: '/bVm6udIB6iKsRqgMdQh6HywuEBj.jpg',
          genre_ids: [53, 28],
          id: 1233069,
          original_language: 'de',
          original_title: 'Exterritorial',
          overview:
            'When her son vanishes inside a US consulate, ex-special forces soldier Sara does everything in her power to find him — and uncovers a dark conspiracy.',
          popularity: 441.0526,
          poster_path: '/jM2uqCZNKbiyStyzXOERpMqAbdx.jpg',
          release_date: '2025-04-29',
          title: 'Exterritorial',
          video: false,
          vote_average: 6.689,
          vote_count: 244,
        },
        {
          adult: false,
          backdrop_path: '/cJvUJEEQ86LSjl4gFLkYpdCJC96.jpg',
          genre_ids: [10752, 28],
          id: 1241436,
          original_language: 'en',
          original_title: 'Warfare',
          overview:
            'A platoon of Navy SEALs embarks on a dangerous mission in Ramadi, Iraq, with the chaos and brotherhood of war retold through their memories of the event.',
          popularity: 467.2245,
          poster_path: '/j8tqBXwH2PxBPzbtO19BTF9Ukbf.jpg',
          release_date: '2025-04-09',
          title: 'Warfare',
          video: false,
          vote_average: 7.16,
          vote_count: 163,
        },
        {
          adult: false,
          backdrop_path: '/rthMuZfFv4fqEU4JVbgSW9wQ8rs.jpg',
          genre_ids: [28, 12, 878],
          id: 986056,
          original_language: 'en',
          original_title: 'Thunderbolts*',
          overview:
            'After finding themselves ensnared in a death trap, seven disillusioned castoffs must embark on a dangerous mission that will force them to confront the darkest corners of their pasts.',
          popularity: 395.8187,
          poster_path: '/m9EtP1Yrzv6v7dMaC9mRaGhd1um.jpg',
          release_date: '2025-04-30',
          title: 'Thunderbolts*',
          video: false,
          vote_average: 7.544,
          vote_count: 663,
        },
        {
          adult: false,
          backdrop_path: '/fTrQsdMS2MUw00RnzH0r3JWHhts.jpg',
          genre_ids: [28, 80, 53],
          id: 1197306,
          original_language: 'en',
          original_title: 'A Working Man',
          overview:
            "Levon Cade left behind a decorated military career in the black ops to live a simple life working construction. But when his boss's daughter, who is like family to him, is taken by human traffickers, his search to bring her home uncovers a world of corruption far greater than he ever could have imagined.",
          popularity: 347.3299,
          poster_path: '/6FRFIogh3zFnVWn7Z6zcYnIbRcX.jpg',
          release_date: '2025-03-26',
          title: 'A Working Man',
          video: false,
          vote_average: 6.506,
          vote_count: 599,
        },
        {
          adult: false,
          backdrop_path: '/4A5HH9HkCPqAwyYL6CnA0mxbYjn.jpg',
          genre_ids: [28, 80, 53],
          id: 1144430,
          original_language: 'fr',
          original_title: 'Balle perdue 3',
          overview:
            'Car genius Lino returns to conclude his vendetta against Areski and the corrupt commander who ruined their lives in this turbo-charged trilogy finale.',
          popularity: 358.4888,
          poster_path: '/qycPITRqXgPai7zj1gKffjCdSB5.jpg',
          release_date: '2025-05-06',
          title: 'Last Bullet',
          video: false,
          vote_average: 7.282,
          vote_count: 39,
        },
        {
          adult: false,
          backdrop_path: '/12tEzU0bNYKIjXXEwI5abuOotHF.jpg',
          genre_ids: [37],
          id: 710258,
          original_language: 'en',
          original_title: 'Rust',
          overview:
            'Infamous outlaw Harland Rust breaks his estranged grandson Lucas out of prison, after Lucas is convicted to hang for an accidental murder. The two must outrun legendary U.S Marshal Wood Helm and bounty hunter Fenton "Preacher" Lang who are hot on their tails. Deeply buried secrets rise from the ashes and an unexpected familial bond begins to form as the mismatched duo tries to survive the merciless American Frontier.',
          popularity: 293.465,
          poster_path: '/tbJ3RkA2s6X5qrBzrYHYTxvDBui.jpg',
          release_date: '2025-05-01',
          title: 'Rust',
          video: false,
          vote_average: 6.3,
          vote_count: 35,
        },
        {
          adult: false,
          backdrop_path: '/2Nti3gYAX513wvhp8IiLL6ZDyOm.jpg',
          genre_ids: [10751, 35, 12, 14],
          id: 950387,
          original_language: 'en',
          original_title: 'A Minecraft Movie',
          overview:
            "Four misfits find themselves struggling with ordinary problems when they are suddenly pulled through a mysterious portal into the Overworld: a bizarre, cubic wonderland that thrives on imagination. To get back home, they'll have to master this world while embarking on a magical quest with an unexpected, expert crafter, Steve.",
          popularity: 271.0468,
          poster_path: '/iPPTGh2OXuIv6d7cwuoPkw8govp.jpg',
          release_date: '2025-03-31',
          title: 'A Minecraft Movie',
          video: false,
          vote_average: 6.161,
          vote_count: 846,
        },
        {
          adult: false,
          backdrop_path: '/65MVgDa6YjSdqzh7YOA04mYkioo.jpg',
          genre_ids: [28, 80, 53],
          id: 668489,
          original_language: 'en',
          original_title: 'Havoc',
          overview:
            "When a drug heist swerves lethally out of control, a jaded cop fights his way through a corrupt city's criminal underworld to save a politician's son.",
          popularity: 209.4306,
          poster_path: '/r46leE6PSzLR3pnVzaxx5Q30yUF.jpg',
          release_date: '2025-04-24',
          title: 'Havoc',
          video: false,
          vote_average: 6.551,
          vote_count: 557,
        },
        {
          adult: false,
          backdrop_path: '/eDBZN0TbWkxoAB0qIDFagVcPPTN.jpg',
          genre_ids: [27, 14, 35, 12],
          id: 1153714,
          original_language: 'en',
          original_title: 'Death of a Unicorn',
          overview:
            'A father and daughter accidentally hit and kill a unicorn while en route to a weekend retreat, where his billionaire boss seeks to exploit the creature’s miraculous curative properties.',
          popularity: 186.0114,
          poster_path: '/lXR32JepFwD1UHkplWqtBP1K79z.jpg',
          release_date: '2025-03-27',
          title: 'Death of a Unicorn',
          video: false,
          vote_average: 6.469,
          vote_count: 176,
        },
        {
          adult: false,
          backdrop_path: '/op3qmNhvwEvyT7UFyPbIfQmKriB.jpg',
          genre_ids: [28, 14, 12],
          id: 324544,
          original_language: 'en',
          original_title: 'In the Lost Lands',
          overview:
            'A queen sends the powerful and feared sorceress Gray Alys to the ghostly wilderness of the Lost Lands in search of a magical power, where she and her guide, the drifter Boyce, must outwit and outfight both man and demon.',
          popularity: 153.6765,
          poster_path: '/dDlfjR7gllmr8HTeN6rfrYhTdwX.jpg',
          release_date: '2025-02-27',
          title: 'In the Lost Lands',
          video: false,
          vote_average: 6.332,
          vote_count: 340,
        },
        {
          adult: false,
          backdrop_path: '/kFXtAwJYbRTqiyQpE9kDc8qEP3X.jpg',
          genre_ids: [28, 53],
          id: 1225915,
          original_language: 'hi',
          original_title: 'ज्वेल थीफ: द हीस्ट बिगिन्स',
          overview:
            'In this high-octane battle of wits and wills, ingenious con artist Rehan devises a diamond heist while trying to outsmart Rajan, his sadistic adversary.',
          popularity: 187.7973,
          poster_path: '/eujLbO0kf1eqWC8XpHUJdtAVW2J.jpg',
          release_date: '2025-04-25',
          title: 'Jewel Thief: The Heist Begins',
          video: false,
          vote_average: 6.557,
          vote_count: 35,
        },
        {
          adult: false,
          backdrop_path: '/svLj9iyRmmeNVRsMreANn7DPPZE.jpg',
          genre_ids: [27],
          id: 1356236,
          original_language: 'en',
          original_title: 'Saint Catherine',
          overview:
            'An orphaned girl is rescued from a satanic ritual and taken to Saint Catherine Institute for homeless youth. There she will learn new skills while facing demons that stalk her.',
          popularity: 137.8021,
          poster_path: '/hBJdzKPeDaC96AzlrtMWBomYSZV.jpg',
          release_date: '2024-09-24',
          title: 'Saint Catherine',
          video: false,
          vote_average: 6.239,
          vote_count: 23,
        },
        {
          adult: false,
          backdrop_path: '/nAxGnGHOsfzufThz20zgmRwKur3.jpg',
          genre_ids: [28, 27, 53],
          id: 1233413,
          original_language: 'en',
          original_title: 'Sinners',
          overview:
            'Trying to leave their troubled lives behind, twin brothers return to their hometown to start again, only to discover that an even greater evil is waiting to welcome them back.',
          popularity: 143.6438,
          poster_path: '/jYfMTSiFFK7ffbY2lay4zyvTkEk.jpg',
          release_date: '2025-04-16',
          title: 'Sinners',
          video: false,
          vote_average: 7.6,
          vote_count: 626,
        },
        {
          adult: false,
          backdrop_path: '/1YMrOtrW7b4pL2lfD8UciZPOJGs.jpg',
          genre_ids: [18, 53, 9648],
          id: 974576,
          original_language: 'en',
          original_title: 'Conclave',
          overview:
            'After the unexpected death of the Pope, Cardinal Lawrence is tasked with managing the covert and ancient ritual of electing a new one. Sequestered in the Vatican with the Catholic Church’s most powerful leaders until the process is complete, Lawrence finds himself at the center of a conspiracy that could lead to its downfall.',
          popularity: 108.6101,
          poster_path: '/l4WXg5oQPK6GVlISKQNIUGb8rKJ.jpg',
          release_date: '2024-10-25',
          title: 'Conclave',
          video: false,
          vote_average: 7.205,
          vote_count: 2371,
        },
        {
          adult: false,
          backdrop_path: '/6dC7ULfiutxwEAs7LjWHL2Tc7Zv.jpg',
          genre_ids: [27, 35],
          id: 1124620,
          original_language: 'en',
          original_title: 'The Monkey',
          overview:
            'When twin brothers find a mysterious wind-up monkey, a series of outrageous deaths tear their family apart. Twenty-five years later, the monkey begins a new killing spree forcing the estranged brothers to confront the cursed toy.',
          popularity: 112.5005,
          poster_path: '/yYa8Onk9ow7ukcnfp2QWVvjWYel.jpg',
          release_date: '2025-02-14',
          title: 'The Monkey',
          video: false,
          vote_average: 5.9,
          vote_count: 586,
        },
        {
          adult: false,
          backdrop_path: '/Zes06xI18u47pblwRGTtqRm0R8.jpg',
          genre_ids: [28, 53, 80, 18],
          id: 1276073,
          original_language: 'ja',
          original_title: '新幹線大爆破',
          overview:
            'When panic erupts on a Tokyo-bound bullet train that will explode if it slows below 100 kph, authorities race against time to save everyone on board.',
          popularity: 95.0194,
          poster_path: '/jmzxlDBwgvRbpPJzNQwizyn9UEn.jpg',
          release_date: '2025-04-23',
          title: 'Bullet Train Explosion',
          video: false,
          vote_average: 6.883,
          vote_count: 154,
        },
        {
          adult: false,
          backdrop_path: '/iznPd7PLnCBA1G50M4DuA9wvCIy.jpg',
          genre_ids: [35],
          id: 1307520,
          original_language: 'es',
          original_title: 'La más fan',
          overview:
            'Canceled in Hollywood, star Lana Cruz heads to her hometown for a comeback. Enter Polly, her fan-turned-frenemy. Can Lana dodge drama and reclaim her fame?',
          popularity: 76.5037,
          poster_path: '/wgUmsekYPOt9ZQ8ero91qRnmhQY.jpg',
          release_date: '2025-05-01',
          title: 'The Biggest Fan',
          video: false,
          vote_average: 5.2,
          vote_count: 15,
        },
        {
          adult: false,
          backdrop_path: '/sNx1A3822kEbqeUxvo5A08o4N7o.jpg',
          genre_ids: [28, 35, 53],
          id: 1195506,
          original_language: 'en',
          original_title: 'Novocaine',
          overview:
            'When the girl of his dreams is kidnapped, everyman Nate turns his inability to feel pain into an unexpected strength in his fight to get her back.',
          popularity: 80.5385,
          poster_path: '/xmMHGz9dVRaMY6rRAlEX4W0Wdhm.jpg',
          release_date: '2025-03-12',
          title: 'Novocaine',
          video: false,
          vote_average: 6.906,
          vote_count: 712,
        },
        {
          adult: false,
          backdrop_path: '/2CcMG34v4PCynvxwmBUwVtNqsKL.jpg',
          genre_ids: [9648, 53],
          id: 1249213,
          original_language: 'en',
          original_title: 'Drop',
          overview:
            'Violet, a widowed mother on her first date in years, arrives at an upscale restaurant where she is relieved that her date, Henry, is more charming and handsome than she expected. But their chemistry begins to curdle as Violet begins being irritated and then terrorized by a series of anonymous drops to her phone.',
          popularity: 83.6773,
          poster_path: '/dS2S5lpfgRIRQOb7LDCjNsQqKjp.jpg',
          release_date: '2025-04-10',
          title: 'Drop',
          video: false,
          vote_average: 6.554,
          vote_count: 271,
        },
        {
          adult: false,
          backdrop_path: '/k32XKMjmXMGeydykD32jfER3BVI.jpg',
          genre_ids: [28, 9648, 18],
          id: 1045938,
          original_language: 'en',
          original_title: 'G20',
          overview:
            'After the G20 Summit is overtaken by terrorists, President Danielle Sutton must bring all her statecraft and military experience to defend her family and her fellow leaders.',
          popularity: 67.8483,
          poster_path: '/wv6oWAleCJZUk5htrGg413t3GCy.jpg',
          release_date: '2025-04-10',
          title: 'G20',
          video: false,
          vote_average: 6.569,
          vote_count: 463,
        },
      ],
      link: 'now_playing',
      isLoading: false,
      title: 'Now Playing',
    };

    const { container } = render(<MoviesSlider {...props} />);
    expect(container).toMatchSnapshot();
  });
});
