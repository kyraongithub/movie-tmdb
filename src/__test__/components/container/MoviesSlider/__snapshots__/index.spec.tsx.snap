// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MoviesSlider > render MoviesSlider component 1`] = `
<div>
  <section
    class="w-full px-4 py-6"
  >
    <div
      class="flex items-center justify-between mb-4"
    >
      <h2
        class="text-xl font-bold  hover:text-primary transition-colors"
      >
        <h1>
          Now Playing
        </h1>
      </h2>
      <a
        class="text-sm text-gray-400 hover:text-gray-600 transition-colors"
        href="/category/now_playing"
      >
        View All
      </a>
    </div>
    <div
      class="relative"
    >
      <div
        class="flex overflow-x-scroll space-x-4 pb-4 scrollbar-hide  overflow-y-hidden"
      >
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Exterritorial"
            src="https://image.tmdb.org/t/p/w500//jM2uqCZNKbiyStyzXOERpMqAbdx.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Exterritorial
            </p>
            <p>
              ⭐
              6.7
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Warfare"
            src="https://image.tmdb.org/t/p/w500//j8tqBXwH2PxBPzbtO19BTF9Ukbf.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Warfare
            </p>
            <p>
              ⭐
              7.2
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Thunderbolts*"
            src="https://image.tmdb.org/t/p/w500//m9EtP1Yrzv6v7dMaC9mRaGhd1um.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Thunderbolts*
            </p>
            <p>
              ⭐
              7.5
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="A Working Man"
            src="https://image.tmdb.org/t/p/w500//6FRFIogh3zFnVWn7Z6zcYnIbRcX.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              A Working Man
            </p>
            <p>
              ⭐
              6.5
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Last Bullet"
            src="https://image.tmdb.org/t/p/w500//qycPITRqXgPai7zj1gKffjCdSB5.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Last Bullet
            </p>
            <p>
              ⭐
              7.3
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Rust"
            src="https://image.tmdb.org/t/p/w500//tbJ3RkA2s6X5qrBzrYHYTxvDBui.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Rust
            </p>
            <p>
              ⭐
              6.3
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="A Minecraft Movie"
            src="https://image.tmdb.org/t/p/w500//iPPTGh2OXuIv6d7cwuoPkw8govp.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              A Minecraft Movie
            </p>
            <p>
              ⭐
              6.2
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Havoc"
            src="https://image.tmdb.org/t/p/w500//r46leE6PSzLR3pnVzaxx5Q30yUF.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Havoc
            </p>
            <p>
              ⭐
              6.6
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Death of a Unicorn"
            src="https://image.tmdb.org/t/p/w500//lXR32JepFwD1UHkplWqtBP1K79z.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Death of a Unicorn
            </p>
            <p>
              ⭐
              6.5
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="In the Lost Lands"
            src="https://image.tmdb.org/t/p/w500//dDlfjR7gllmr8HTeN6rfrYhTdwX.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              In the Lost Lands
            </p>
            <p>
              ⭐
              6.3
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Jewel Thief: The Heist Begins"
            src="https://image.tmdb.org/t/p/w500//eujLbO0kf1eqWC8XpHUJdtAVW2J.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Jewel Thief: The Heist Begins
            </p>
            <p>
              ⭐
              6.6
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Saint Catherine"
            src="https://image.tmdb.org/t/p/w500//hBJdzKPeDaC96AzlrtMWBomYSZV.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Saint Catherine
            </p>
            <p>
              ⭐
              6.2
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Sinners"
            src="https://image.tmdb.org/t/p/w500//jYfMTSiFFK7ffbY2lay4zyvTkEk.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Sinners
            </p>
            <p>
              ⭐
              7.6
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Conclave"
            src="https://image.tmdb.org/t/p/w500//l4WXg5oQPK6GVlISKQNIUGb8rKJ.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Conclave
            </p>
            <p>
              ⭐
              7.2
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="The Monkey"
            src="https://image.tmdb.org/t/p/w500//yYa8Onk9ow7ukcnfp2QWVvjWYel.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              The Monkey
            </p>
            <p>
              ⭐
              5.9
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Bullet Train Explosion"
            src="https://image.tmdb.org/t/p/w500//jmzxlDBwgvRbpPJzNQwizyn9UEn.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Bullet Train Explosion
            </p>
            <p>
              ⭐
              6.9
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="The Biggest Fan"
            src="https://image.tmdb.org/t/p/w500//wgUmsekYPOt9ZQ8ero91qRnmhQY.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              The Biggest Fan
            </p>
            <p>
              ⭐
              5.2
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Novocaine"
            src="https://image.tmdb.org/t/p/w500//xmMHGz9dVRaMY6rRAlEX4W0Wdhm.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Novocaine
            </p>
            <p>
              ⭐
              6.9
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="Drop"
            src="https://image.tmdb.org/t/p/w500//dS2S5lpfgRIRQOb7LDCjNsQqKjp.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              Drop
            </p>
            <p>
              ⭐
              6.6
            </p>
          </div>
        </div>
        <div
          class="cursor-pointer hover:scale-110 transition-all min-w-[200px]"
        >
          <img
            alt="G20"
            src="https://image.tmdb.org/t/p/w500//wv6oWAleCJZUk5htrGg413t3GCy.jpg"
          />
          <div
            class="flex justify-between"
          >
            <p>
              G20
            </p>
            <p>
              ⭐
              6.6
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
